manifests/p/PuTTY-CAC/PuTTY-CAC/0.76u4/PuTTY-CAC.PuTTY-CAC.locale.en-US.yaml

```
@@ -0,0 +1,31 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: PuTTY-CAC.PuTTY-CAC
+PackageVersion: 0.76u4
+PackageLocale: en-US
+Publisher: Bryan Berns

```

```suggestion
Publisher: Simon Tatham
```
This must match Control Panel.